OutOfSchool/OutOfSchool.DataAccess/Repository/IAboutPortalRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository
+{
+    public interface IAboutPortalRepository : ISensitiveEntityRepository<AboutPortal>
+    {
+        IUnitOfWork UnitOfWork { get; }
```

Comment:
Don't see if `UnitOfWork` is used anywhere. Probably remove it.

Reply:
Removed it.